metaserver/src/test/scala/scala/meta/languageserver/ctags/ClasspathCtagsTest.scala

```diff
@@ -0,0 +1,268 @@
+package scala.meta.languageserver.ctags
+
+import java.nio.file.Paths
+import scala.meta.languageserver.Jars
+import scala.meta.testkit.DiffAssertions
+import org.scalatest.FunSuite
+
+class ClasspathCtagsTest extends FunSuite with DiffAssertions {
+
+  // NOTE(olafur) this test is a bit slow since it downloads jars from the internet.
+  test("index classpath") {
```

Comment:
should we group this under a `ci-slow` target to run separately (like we do in Scalafix?)

Reply:
I like the idea of a CI slow category as well.